llpc/lower/llpcSpirvLowerUtil.h

```diff
@@ -0,0 +1,67 @@
+/*
+ ***********************************************************************************************************************
+ *
+ *  Copyright (c) 2020 Advanced Micro Devices, Inc. All Rights Reserved.
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in all
+ *  copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ *
+ **********************************************************************************************************************/
+/**
+ ***********************************************************************************************************************
+ * @file  llpcSpirvLowerUtil.h
+ * @brief LLPC header file: utilities for use by LLPC front-end
+ ***********************************************************************************************************************
+ */
+#pragma once
+
+#include "llpc.h"
+
+namespace llvm
+{
+
+class Function;
+class Module;
+
+} // llvm
+
+namespace Llpc
+{
+
+using namespace llvm;
+
+// Well-known names in the front-end.
+namespace LlpcName
+{
+
+const static char GlobalProxyPrefix[]             = "__llpc_global_proxy_";
```

Comment:
Consider using `llvm::StringLiteral` for these constants.

Reply:
As this code was just moved from somewhere else, I think I'll leave it as is for now.